### ISSUE TITLE
Depth tab and scripts not showing up

### ISSUE THREAD

**Issue by Wolfven7** [thygate/stable-diffusion-webui-depthmap-script#275]
Hello!

I've searched around the closed and open issues about the depth tab not showing up, however the fixes havent worked and the problems are from different setups (mac, collab, etc).

I am running A1111 locally on Windows using my 3090. I had the Depth extension working some time in the past. At some point I deleted A111 (needed driver space). After some time I reinstalled and hoped to get Depth back up and running. Its not going well.

I download it through the UI as most extensions, copying and pasting the git link into the "download from URL" box. This works. I see the depth extension is checked off and updated.

However, it wont show up in any of the tabs and none of the scripts show. I tried to refresh the ui, completely restart the console, reinstall the extension after deleting the extension file, and i'm out of ideas.

if you need more info let me know. 

thanks.

Edit: I've seen somewhere that there are a few requirements that are installed every time the console is launched. the individual restarted the webui-user.bat and noticed them show up after an install. however, i only ever get two no matter how much I restart:

Installing imageio-ffmpeg requirement for depthmap script
Installing pyqt5 requirement for depthmap script

hope this helps.

**Comment by thygate** [comment]
please post console output

**Comment by Wolfven7** [comment]
venv "C:\stable-diffusion-webui-master\stable-diffusion-webui-master\venv\Scripts\Python.exe"
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Commit hash: <none>
Installing requirements for Web UI

Installing imageio-ffmpeg requirement for depthmap script
Installing pyqt5 requirement for depthmap script

Installing requirements for Unprompted - natural language processing: pattern@git+https://github.com/NicolasBizzozzero/pattern
Installing requirements for Unprompted - [img2pez]: sentence-transformers
Installing requirements for Unprompted - [pix2pix_zero]: salesforce-lavis
Installing requirements for Unprompted - [zoom_enhance]: color-matcher

Launching Web UI with arguments: --xformers --theme dark
2023-07-03 23:41:48,929 - ControlNet - INFO - ControlNet v1.1.228
ControlNet preprocessor location: C:\stable-diffusion-webui-master\stable-diffusion-webui-master\extensions\sd-webui-controlnet\annotator\downloads
2023-07-03 23:41:49,094 - ControlNet - INFO - ControlNet v1.1.228
Error loading script: depthmap.py
Traceback (most recent call last):
  File "C:\stable-diffusion-webui-master\stable-diffusion-webui-master\modules\scripts.py", line 248, in load_scripts
    script_module = script_loading.load_module(scriptfile.path)
  File "C:\stable-diffusion-webui-master\stable-diffusion-webui-master\modules\script_loading.py", line 11, in load_module
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "C:\stable-diffusion-webui-master\stable-diffusion-webui-master\extensions\stable-diffusion-webui-depthmap-script\scripts\depthmap.py", line 56, in <module>
    from dmidas.dpt_depth import DPTDepthModel
  File "C:\stable-diffusion-webui-master\stable-diffusion-webui-master\extensions\stable-diffusion-webui-depthmap-script\dmidas\dpt_depth.py", line 5, in <module>
    from .blocks import (
  File "C:\stable-diffusion-webui-master\stable-diffusion-webui-master\extensions\stable-diffusion-webui-depthmap-script\dmidas\blocks.py", line 4, in <module>
    from .backbones.beit import (
  File "C:\stable-diffusion-webui-master\stable-diffusion-webui-master\extensions\stable-diffusion-webui-depthmap-script\dmidas\backbones\beit.py", line 9, in <module>
    from timm.models.beit import gen_relative_position_index
ModuleNotFoundError: No module named 'timm.models.beit'

(SETUP) Loading Unprompted v9.5.0 by Therefore Games
(SETUP) Initializing Unprompted object...
(SETUP) Loading configuration files...
(SETUP) Logging enabled for the following message types: RESULT,WARNING,ERROR
Loading weights [44fa3d957e] from C:\stable-diffusion-webui-master\stable-diffusion-webui-master\models\Stable-diffusion\fluffyrock_3mE57OffsetNoiseE42.safetensors
Creating model from config: C:\stable-diffusion-webui-master\stable-diffusion-webui-master\configs\v1-inference.yaml
LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.52 M params.
Applying xformers cross attention optimization.
Textual inversion embeddings loaded(0):
Model loaded in 2.7s (create model: 0.3s, apply weights to model: 0.5s, apply half(): 0.5s, move model to device: 0.5s, load textual inversion embeddings: 0.9s).
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Startup time: 14.0s (import gradio: 2.7s, import ldm: 1.5s, other imports: 2.8s, list extensions: 0.5s, list SD models: 0.2s, setup codeformer: 0.1s, load scripts: 2.0s, load SD checkpoint: 3.0s, create ui: 0.7s, gradio launch: 0.3s).

**Comment by Wolfven7** [comment]
Hey, just figured out what to do after following some steps in this issue here: https://github.com/thygate/stable-diffusion-webui-depthmap-script/issues/169

I first did "pip install torchtext" in command prompt (turns out I didnt have it). then did "pip install --force-reinstall timm==0.6.12"

I still had the same issue, however.

after that I removed the extension "unprompted" (https://github.com/ThereforeGames/unprompted)

everything works now.
